hw4/index.js

```diff
@@ -0,0 +1,13 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+    const firstNumber = prompt("Enter 1st number");
```

Comment:
it will be better to show how You change ```prompt``` type to Number type on the init step
```suggestion
    const firstNumber = Number(prompt("Enter 1st number"));
```
OR
```suggestion
    const firstNumber = +prompt("Enter 1st number");
```
1) It would avoid any errors in the future
2) It's easier to understand
3) You will avoid redundant type changing at every step (++, -, /, *)